packages/vats/src/types.js

```diff
@@ -38,11 +38,17 @@
  * @property {(key: string) => void} reserve Mark a key as reserved; will
  * return a promise that is fulfilled when the key is updated (or rejected when
  * deleted).
+ * @property {( key: string, newValue: unknown, newAdmin?: unknown) =>
+ *   Promise<any> } default Update if not already updated.  Return
+ *   existing value, or newValue if not existing.
  * @property {(
- *   key: string, newValue: unknown, newAdmin?: unknown) => void
+ *   key: string, newValue: unknown, newAdmin?: unknown) => Promise<any>
+ * } set Update only if already initialized. Reject if not.
```

Comment:
I don't see this used anywhere. Are we going to need it?

Reply:
I put it in for symmetry with the stores.